constellation-app/constellation
# Hide Data Access Categories

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change

A new feature is added to hide data access categories that are not used by the users. 

### Alternate Designs



### Why Should This Be In Core?

Core functionality

### Benefits

Users can filter the data access view to display only the categories they want  

### Possible Drawbacks

Performance

### Verification Process

1. Open Constellation and go to Setup -> Options -> Constellation and select the new Data Access View tab
2. Move any category from visible list to hidden list and click apply
3. Close and open Constellation
4. Check to verify the hidden category is removed from the DAV   

### Applicable Issues
